package.json

```diff
@@ -117,6 +117,7 @@
     "invariant": "^2.2.2",
     "isomorphic-ws": "^4.0.1",
     "json-rpc-2.0": "^0.2.18",
+    "ledger-live-platform-sdk": "git+ssh://git@github.com:LedgerHQ/ledger-live-platform-sdk.git#add-serializeAccount",
```

Comment:
For now the dependence is a git repo because no package has been published yet (the SDK is still actively WIP) but obviously once a first version is published, it will be handled as any other packaged dep